truss/templates/server/common/patches/transformers/patch.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+from joblib import Parallel, delayed
+from transformers.utils import hub
+
+
+def get_checkpoint_shard_files(
```

Comment:
Patches are especially important to document from maintainability pov. Can we add a descriptive doc comment here about what the patch does, how, and what that achieves? Also, the key difference from the original implementation. Same for the other patch.

Reply:
+1 - maybe we can have a comment or readme.md per patch in the `patches/` folder